.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Train recipe '....'
+2. Change param '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - OS [e.g. uname -s -r -m]
+ - Relevant HW info, GPU [e.g. nvidia-smi]
+ - Super Gradients version
+
```

Comment:
I would like to see information about the python libraries installed - like running pip [freeze](https://pip.pypa.io/en/stable/cli/pip_freeze/) command. This way we can reproduce the bug in exact environment.